transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SchemaManager.java

```diff
@@ -95,12 +104,24 @@
   private static final SchemaManager.Migration MIGRATE_TO_V3 =
       db -> db.execSQL("ALTER TABLE events ADD COLUMN payload_encoding TEXT");
 
+  private static final SchemaManager.Migration MIGRATE_TO_V4 =
+      db -> {
+        db.execSQL("ALTER TABLE events ADD COLUMN inline BOOLEAN DEFAULT 1");
```

Comment:
What happens to existing inline events that may be too large. Should we cleanup since migration seems impossible ?

Reply:
I think it's highly unlikely that we have such events out there and the cleanup would be pretty involved if we don't want to lose data, we would have to:

* query all events with len(payload) > X
* for each event select X bytes chunk by chunk and put into new table
* clear inline + payload fields

And all of this would have to be maintained forever.

So given how unlikely it is that we have large payloads out there, I'd prefer not to bother with this. wdyt?